src/ctim/schemas/ttp.clj

```diff
@@ -106,12 +106,11 @@
                     (str
                      "specifies other TTPs asserted to be related to this cyber"
                      " threat TTP"))
-     :source (describe s/Str "source of this cyber threat TTP")})
+     :source (describe s/Str "source of this cyber threat TTP")
+     :kill_chains [v/KillChain]})
```

Comment:
It might get tricky. In some cases, the set of servers (identified by IPs)  of same malware family might be used/mapped to different phases of KC. for example, few ip addresses in same set could be CnC and few could be used to upload exfil data (Action phase ) , few could serve initial malware payloads (delivery). So assigning a collection of IPaddresses (indicators) to particular phase of KC. So ideally one or more KC phases are associated with a TTP.  (Similar to STIX - [TTP](http://stixproject.github.io/data-model/1.2/ttp/TTPType/) abstraction ) .  So in current modelling if it is conveying the plurality of distinct KC phases per TTP then we are good.